packages/expo-image-picker/ios/ImagePickerDelegate.swift

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+/**
+ Protocol that describes scenarios we care about while the user is picking media.
+ */
+protocol OnMediaPickingResultHandler {
+  func didPickedMedia(mediaInfo: MediaInfo) -> Void
+  func didCancelPicking() -> Void
```

Comment:
Nitpick, but these names are inconsistent - `did picked media` is using past form twice 😉 

```suggestion
  func didPickMedia(mediaInfo: MediaInfo) -> Void
  func didCancelPicking() -> Void
```